docs/how-to-guides/client-supplied-encryption.md

```diff
@@ -0,0 +1,72 @@
+# Client Supplied Encryption
+
+PKI (Public Key Infrastructure) encryption technology is leveraged to provide additional security for customers using the VoiceBase platform. The customer provides us with their PGP "public key", which can be used to encrypt the data, but not to decrypt it. The customer keeps their "private key" and never sends it to VoiceBase. When the media is processed, we encrypt the results using the customer's "public key" for storage and delivery to the customer. The customer receives the encrypted data, and uses the "private key" to decrypt and use the results.
+
+Users may create a PGP key using PGP or GPG tools. We recommend to use an "RSA and RSA" key with 2048 bits, with one-year crypto-period.
+
+### Upload the public key to VoiceBase
+
+```bash
+curl -s "https://apis.voicebase.com/v3/security/public-keys"
+--header "Authorization: Bearer XXXXXX"
```

Comment:
@Daniel085 Let's be consistent about Authorization: Bearer $TOKEN instead of XXXXXXX